### PR TITLE
Use Sass @use instead of @import

### PR DESCRIPTION
**Pull request by ldewald**

### CODE REVIEW

**Review by stephenjwatkins — COMMENTED** [review]
nice! looks good overall. just one question:

i don't have a super strong opinion about this but i'm wondering if for our "common" helpers in our `.module.scss` files if we should `@use "../styles/common" as *`. i don't think the `c.design-token()`, etc is too bad, but i wonder if it may make sense to directly expose the helpers forwarded in common to make it slightly easier to use 🤔 curious what you / others think?

**Comment by ldewald** [comment]
> nice! looks good overall. just one question:
> 
> i don't have a super strong opinion about this but i'm wondering if for our "common" helpers in our `.module.scss` files if we should `@use "../styles/common" as *`. i don't think the `c.design-token()`, etc is too bad, but i wonder if it may make sense to directly expose the helpers forwarded in common to make it slightly easier to use 🤔 curious what you / others think?

I wondered the same thing and went back and forth myself so updated with this pattern instead

File: easy-ui-react/src/Button/Button.module.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@use "./mixins" as Button;
-@import "../styles/common";
+@use "../styles/common" as *;
```
Comment (ldewald) [inline easy-ui-react/src/Button/Button.module.scss:1]: Swapped the order because it felt better to have it come before the namespaced `@use`

Comment (stephenjwatkins) [reply easy-ui-react/src/Button/Button.module.scss:1]: agreed!

**Review by stephenjwatkins — APPROVED** [review]
nice!

**Pull request merged**